CHANGELOG.md

```diff
@@ -4,14 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v4.0.0](https://github.com/opus-codium/puppet-odoo/tree/v4.0.0) (2024-04-12)
+## [v4.1.0](https://github.com/opus-codium/puppet-odoo/tree/v4.1.0) (2024-06-06)
```

Comment:
Huh! Somebody merged #76 but forgot to run `rake release`.  Pointy hat to @neomilium